apps/api/.env.example

```
@@ -1,15 +1,34 @@
 GOOGLE_LOGO_BUCKET_PROJECT_ID=****
-NEXT_PUBLIC_API_URL=****
-NEXT_PUBLIC_AUTH_URL=****
-NEXT_PUBLIC_CHANNEL=****
-NEXT_PUBLIC_MAP_URL=****
\ No newline at end of file
+
+# -- Client-side URLs (Next.js public vars) -----------------------------------
+NEXT_PUBLIC_API_URL=http://localhost:3001
+NEXT_PUBLIC_AUTH_URL=http://localhost:3004
+NEXT_PUBLIC_MAP_URL=http://localhost:3000
+NEXT_PUBLIC_CHANNEL=local
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)